demo/api/src/footer/footer.module.ts

```diff
@@ -9,6 +9,6 @@ import { FootersService } from "./generated/footers.service";
 
 @Module({
     imports: [MikroOrmModule.forFeature([Footer, FooterContentScope])],
-    providers: [FooterResolver, FootersService, DependenciesResolverFactory.create(Footer)],
+    providers: [FooterResolver, FootersService, DependenciesResolverFactory.create({ classRef: Footer, requiredPermission: "pageTree" })],
```

Comment:
I'm not sure about `classRef`. Do you have an idea for a better name?

Reply:
I've thought about entity too, but I thought that it may not be an entity. However, since the generic extends `AnyEntity`, I believe it's safe to assume that it will always be an entity. Changed: afd47e9a3c4ed44e69c1fde344cf42982cada35d